src/deluge/io/debug/sysex.h

```diff
@@ -32,3 +32,22 @@ void loadCheckAndRun(uint8_t* data, int32_t len);
 #endif
 
 } // namespace Debug
+
+namespace SysEx {
```

Comment:
This is in a weird spot - could you put them in midi engine or somewhere else in the midi folder?

Reply:
moved to io/midi